tools/cli/README.md

```diff
@@ -0,0 +1,24 @@
+# Development tools
+
+### Swift development
+
+**Problem**
+
+Native `macos` and `ios` projects share most of the code. Sadly, CocoaPods does not allow to
+reference files outside of the root project directory, or to symlink `.swift` files.
```

Comment:
Really? The symlink solution does at least work for objective C. For swift, I have seen other solutions to code sharing, though, a type of include/import-based solution.